### PR TITLE
Removes the Emag functionality from the Goat Plushie

### PR DESCRIPTION
**Pull request by RiskySikh**
## About The Pull Request

The purpose of this PR was to fix the throw force of the goat plushie which was affected by the emagging of the plushie. Which is why removing the regarding emagging fixes the throwforce. 

## Why It's Good For The Game

It fixes's the throwforce issue of the goat plushie, since emagging it reduces the throwforce. Which is why removing the emag code is crucial.

## Changelog
:cl:
del: Emagged Goat Plushie which fixes throwforce.
/:cl:

### CODE REVIEW

**Comment by yeeyeh** [comment]
good work, proud of you bindy johal

**Comment by tralezab** [comment]
good work, proud of you jindy bohal

**Comment by nemvar** [comment]
ripperoni
RIP

**Review by ExcessiveUseOfCobblestone — DISMISSED** [review]

**Comment by MrDoomBringer** [comment]
free bindy

**Comment by Shadowflame909** [comment]
>good first pr
>Not yeah i'm a coder what about it like me

Good first PR RiskySikh!

**Comment by LemonInTheDark** [comment]
:)

You may want to add a space before your name in the CL, it can be fucky, at least in my experience.

**Comment by RiskySikh** [comment]
thank you very much guys for the support

**Comment by RiskySikh** [comment]
> 
> 
> :)
> 
> You may want to add a space before your name in the CL, it can be fucky, at least in my experience.

my bad

**Comment by Whoneedspacee** [comment]
im pretty sure emagging this was supposed to make it deal less damage but attack faster since this changed the cooldown modifier var but did nothing since process only fired every couple seconds or so

emagging plushies is stupid anyways

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by Tharcoonvagh** [comment]
Jv7nQkx

**Comment by Neo-0** [comment]
Disabling this prevents other things that need this interaction from working.
You can just remove the throwforce and modifier thing/var.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
https://github.com/tgstation/tgstation/pull/49950/files#diff-c69f4cb56976195ea90c30f150ba9430R576

remove emagged bit

**Comment by RiskySikh** [comment]
> 
> 
> https://github.com/tgstation/tgstation/pull/49950/files#diff-c69f4cb56976195ea90c30f150ba9430R576
> 
> remove emagged bit

I thought I removed it

**Comment by RiskySikh** [comment]
> 
> 
> Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.
> 
> Your current title and or body content does not meet requirements, so please update it.

understood

**Comment by RiskySikh** [comment]
> 
> 
> https://github.com/tgstation/tgstation/pull/49950/files#diff-c69f4cb56976195ea90c30f150ba9430R576
> 
> remove emagged bit

fixed it

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -573,22 +573,12 @@
 		ram()
 
 /obj/item/toy/plush/goatplushie/angry/proc/ram()
-	if(prob((obj_flags & EMAGGED) ? 98:90) && isturf(loc) && considered_alive(target.mind) && !faction_check(list("goat"), target.faction, FALSE))
```
Comment (Fikou) [inline code/game/objects/items/plushes.dm]: dont remove this, only replace prob((obj_flags & EMAGGED) ? 98:90) with prob(90)

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/plushes.dm]: ^

Comment (RiskySikh) [reply code/game/objects/items/plushes.dm]: understood

Comment (RiskySikh) [reply code/game/objects/items/plushes.dm]: I fixed it

File: code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -573,22 +573,13 @@
...
+if(prob(90) && isturf(loc) && considered_alive(target.mind) && !faction_check(list("goat"), target.faction, FALSE))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/plushes.dm]:
```suggestion
    if(prob(90) && isturf(loc) && considered_alive(target.mind) && !faction_check(list("goat"), target.faction, FALSE))
```

**Comment by RiskySikh** [comment]
uh I keep trying to fix it still doesn't work

**Pull request closed without merging**